book/chapters/chapter5/advanced_tuning_methods_and_black_box_optimization.qmd

```diff
@@ -312,9 +312,8 @@ learner$param_set$set_values(
 )
 ```
 
-We now construct the tuning instance and tuner.
-We use `trm("none")` because Hyperband terminates itself after all brackets have been evaluated.
-We also set `eta = 2` and control the number of repetitions with `repetitions = 1`.
+We now construct the tuning instance and a hyperband tuner with `eta = 2`.
+We use `trm("none")` and set the `repetitions` control parameter to `1`, causing Hyperband to terminates itself after all brackets have been evaluated a single time.
```

Comment:
```suggestion
We use `trm("none")` and set the `repetitions` control parameter to `1` so that Hyperband can terminate itself after all brackets have been evaluated a single time.
```